symbols.xml

```diff
@@ -1220,8 +1224,8 @@
         <global-address name='process_dig' value='0x141026a03'/>
         <global-address name='cursor' value='0x140e60058'/>
         <global-address name='selection_rect' value='0x140e60068'/>
-        <global-address name='gamemode' value='0x140e60088'/>
-        <global-address name='gametype' value='0x140e6008c'/> -4
+        <global-address name='gamemode' value='0x140e6008c'/>
+        <global-address name='gametype' value='0x140e60088'/>
```

Comment:
8 byte data structures will have 8 byte alignment boundaries I believe so it happens that Win32 build has 4 extra padding bytes before the print_time array that is not there on Win64. I verified that all is well if I write code that emits alignment decls and both my win32 and win64 work now. (also added explicit padding after display_frames and original_rect)
long rect_id;
__declspec(align(8)) long print_time[200];
long print_index;

I tried the following but had issues with getting both builds working in menus but should work in theory and would be a better solution. There may have been issues with the padding with other members.
long rect_id;
long long print_time[100];
long print_index;